src/tools/rados/rados.cc

```diff
@@ -1428,7 +1435,10 @@ static int do_get_inconsistent_cmd(const std::vector<const char*> &nargs,
     }
     items.clear();
   }
-  formatter.flush(cout);
+  if (opened) {
+    formatter.close_section();
```

Comment:
oh, i see. but can we move this line to https://github.com/ceph/ceph/pull/7289/files#diff-c24acfa82f15c38ae9370cab8630a579R1479 ? so that we can do the close at the same  place?